oximeter/types/src/schema.rs

```diff
@@ -6,9 +6,6 @@
 
```

Comment:
I don't feel super strongly about it, but I sort of expected this whole file to be in `oximeter-schema`, and possibly re-exported here if needed. What's the rationale for which types are here and which are in `oximeter-schema`?

Reply:
Hmm -- well, `oximeter-schema` depends on `oximeter-types`. Do you mean exporting everything in this file in `oximeter-schema`?

I was imagining `oximeter-types` containing the actual schema types while `oximeter-schema` contains the utilities for working with those types.

Reply:
Going to land this for now -- happy to iterate on it if you feel particularly strongly. Personally I tend to avoid too many re-exports because rust-analyzer often gets confused about which version is canonical, but I see the argument either way as well.

Reply:
> I was imagining oximeter-types containing the actual schema types while oximeter-schema contains the utilities for working with those types.

That's all I was looking for! Just a rule of thumb to help keep the types organized the way you've done. Thanks!